### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (448)
Added two kinds of snackbar. A normal notification and an error notification.

### CODE REVIEW

File: components/text-utilities/text-uti.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    padding: 20px;
+    font-size: 19px;
+    color: var(--primary-color);
+}
+.font-fam-name{
+    color: var(--font-color)
+}
+.different-texts{
+    display: flex;
+    flex-basis: 400px;
```
Comment (amarphule) [inline components/text-utilities/text-uti.css]: you can use `rem` instead `px`.

File: components/text-utilities/text-uti.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+}
+.xxsmall-text{
+    font-size: xx-small;
+}
+.large-gray-text{
+    color: gray;
+    font-size: large;
+}
+.xlarge-gray-text{
+    color: rgb(95, 95, 95);
```
Comment (amarphule) [inline components/text-utilities/text-uti.css]: as you following variables for colors, here also you can create one variable and use it

File: document-css/doc.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+.intro-head, .install-head{
+    color: var(--font-color);
+    font-size: 1.7rem;
+
+}
+.intro-head{
+    margin: -5px;
+}
+.install-head{
+    margin-top: 30px;
```
Comment (amarphule) [inline document-css/doc.css]: for unit use` rem` as you are using it everywhere

File: landing-page/style.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    font-size: 1.1rem;
+    font-family: 'Montserrat', sans-serif;
+}
+.primary-btn{
+    background-color: var(--primary-color);
+    border-radius: 5px;
+    color: var(--background-color);
+    /* opacity: 0.5; */
+}
+.primary-btn:hover{
```
Comment (amarphule) [inline landing-page/style.css]: remove commented code

File: assets/common-css/sidebar.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.sidebar{
+    text-align:center;
+    /*  */
+}
+.doc-topic, .components{
+    font-size: 1.2rem;
+    color: var(--font-color);
+    /* text-decoration-line: underline; */
+}
```
Comment (amarphule) [inline assets/common-css/sidebar.css]: remove commented code

File: assets/common-css/navbar.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
+
+.navbar-container{
+    padding: 20px;
+    /* display: flex; */
```
Comment (Hamdrive) [inline assets/common-css/navbar.css]: Cleanup commented out code

File: assets/common-css/sidebar.css [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.sidebar{
+    text-align:center;
+    /*  */
+}
```
Comment (Hamdrive) [inline assets/common-css/sidebar.css]: Code formatting could be done better

File: components/alert/alert.css [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+    color: var(--alert-color);
+    border: 1px solid var(--alert-color);
+}
+.warning-border{
+    color: var(--warning-color);
+    border: 1px solid var(--warning-color);
+
+}
+
+
```
Comment (Hamdrive) [inline components/alert/alert.css]: code formatting to be done

**Review by amarphule — CHANGES_REQUESTED** [review]
Your code looks good. 
Just one suggestion for your consistency,  as you are following `rem` units use it everywhere. I saw some of the places you used `px`.
Remove commented code to keep file neat and clean.

File: components/snackbar/snackbar.css [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+.retry,
+.cross-btn {
+    border: none;
+    background-color: rgb(66, 65, 65);
+    font-weight: bold;
+    color: #bb86fc;
+}
+
+.toast {
```
Comment (Hamdrive) [inline components/snackbar/snackbar.css]: advise to stick to one name only, dont switch between toast and snackbar classnames

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Maximum only two components should be part of one branch and PR
Also suggest to change the name and description/ demo images of the PR

File: components/text-utilities/text-uti.css (hunk 1 above)
Comment (Jia-A) [reply components/text-utilities/text-uti.css]: okay, forgot to change it here.

File: components/text-utilities/text-uti.css (hunk 2 above)
Comment (Jia-A) [reply components/text-utilities/text-uti.css]: But I have used this color here only that is why I didn't make a variable for it.

File: document-css/doc.css (hunk 3 above)
Comment (Jia-A) [reply document-css/doc.css]: Yep, changing it right away.

File: landing-page/style.css (hunk 4 above)
Comment (Jia-A) [reply landing-page/style.css]: Yes, forgot to format it.

File: assets/common-css/sidebar.css (hunk 5 above)
Comment (Jia-A) [reply assets/common-css/sidebar.css]: Yes

File: components/alert/alert.css (hunk 8 above)
Comment (Jia-A) [reply components/alert/alert.css]: Yes.

File: components/snackbar/snackbar.css (hunk 9 above)
Comment (Jia-A) [reply components/snackbar/snackbar.css]: Okay, changing the name.

**Pull request merged**